packages/test/test-gc-sweep-tests/src/dataObjectWithChildDataObject.ts

```diff
@@ -26,39 +23,40 @@ export class RootDataObjectWithChildDataObject extends DataObjectWithCounter {
     protected async hasInitialized(): Promise<void> {
         await super.hasInitialized();
         await this.createAndReferenceChild();
-        // TODO: Start logic goes here after we agree on what it should be.
+        this.start();
     }
 
-    // TODO: Make private - left public so the build passes
-    public async unreferenceChild() {
+    private unreferenceChild() {
         assert(this.child !== undefined, `Child should be defined when unreferencing!`);
-        this.unreferencedChild = this.child;
         this.child.stop();
         this.root.delete(this.childKey);
-        this.unrefTimeStampMs = Date.now();
     }
 
     private async createAndReferenceChild() {
         this.child = await dataObjectWithCounterFactory.createInstance(this.context.containerRuntime);
-        await this.referenceChild(this.child.handle);
+        this.root.set(this.childKey, this.child.handle);
+        this.child.start();
     }
 
-    private async referenceChild(handle: IFluidHandle<DataObjectWithCounter>) {
-        this.unreferencedChild = this.child;
-        this.root.set(this.childKey, handle);
-    }
+    protected async sendOps() {
+        assert(this.isRunning === true, "Should be running to send ops");
+        while (this.isRunning && !this.disposed) {
+            let opsPerformed = 0;
+            while (opsPerformed < this.opsToWait && this.isRunning && !this.disposed) {
+                super.sendOp();
```

Comment:
It should just increment the counter here. Calling super. here is just making the code less readable and adding overhead.